src/tests/getUnsubscribedURL.test.ts

```diff
@@ -0,0 +1,23 @@
+import getUnsubscribeURL from '../utils/getUnsubscribeURL';
+
+describe('getUnsubscribeURL', () => {
+  test('should return default mail unsubscribe URL', () => {
+    const email = 'test@example.com';
+    const unsubscribeURL = getUnsubscribeURL(email);
+
+    expect(unsubscribeURL).toEqual(
+      `http://localhost:8000/unsubscribe?email=${email}`,
+    );
+  });
+
+  test('should return URL with the passed host and port', () => {
+    const email = 'test@example.com';
+    const host = 'test.com';
+    const port = 5000;
+    const unsubscribeURL = getUnsubscribeURL(email, host, port);
+
+    expect(unsubscribeURL).toEqual(
+      `http://${host}:${port}/unsubscribe?email=${email}`,
+    );
```

Comment:
since you test same function woudn't it be a good idea to write one test that result equals to `http://${host}:${port}/unsubscribe?email=${email}`, ?

Reply:
Well, we were supposed to write tests with edge cases ¯\_(⊙︿⊙)_/¯
And this function has some non-required arguments